cluster-up/cluster/okd-4.1/provider.sh

```diff
@@ -36,13 +43,23 @@ function up() {
         params=" --ocp-console-port $OKD_CONSOLE_PORT ${params}"
     fi
 
+    if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
+        mkdir -p /tmp/pack8s
```

Comment:
I'd use `mktemp` (with a suitable prefix, see man page), mostly to reduce the risk of the setup failing for existing directory. The other reason is for added security, but I don't think we need much security in this case.